internal/boot/bootstrap_termus.go

```diff
@@ -0,0 +1,59 @@
+//go:build termus
```

Comment:
How about just renaming this package bootstrap?

Then we could expose:

bootstrap.Validate()
bootstrap.Run()

Reply:
I suggest `bootstrap.Validate()` and `bootstrap.Do()` (since you don't "run the bootstrap", but you "do" the bootstrapping or "perform" the bootstrapping).